### PR TITLE
Release notes for plugin publishing 1.2.0

### PR DESCRIPTION
**Pull request by ov7a**
refs #23908

### CODE REVIEW

File: subprojects/docs/src/docs/userguide/developing-plugins/publishing_gradle_plugins.adoc [hunk 1]
```diff
@@ -155,6 +155,14 @@ Publish the plugin by using the `publishPlugin` task.
 $ ./gradlew publishPlugins
 ----
 
+You can validate your plugins before actually publishing them using `--validate-only` flag:
```
Comment (jbartok) [inline subprojects/docs/src/docs/userguide/developing-plugins/publishing_gradle_plugins.adoc:158]:
```suggestion
You can validate your plugins before actually publishing them using the `--validate-only` flag:
```

**Review by jbartok — APPROVED** [review]

File: subprojects/docs/src/docs/release/notes.md [hunk 2]
```diff
@@ -330,6 +330,25 @@ In other types of projects, it will use the same version of Java used by the Gra
 
 For more details, see the [user manual](userguide/codenarc_plugin.html).
 
+### Plugin publishing
```
Comment (ljacomet) [inline subprojects/docs/src/docs/release/notes.md:333]:
```suggestion
### Plugin publishing to the Gradle Plugin Portal
```

File: subprojects/docs/src/docs/release/notes.md [hunk 3]
```diff
@@ -330,6 +330,25 @@ In other types of projects, it will use the same version of Java used by the Gra
...
+
```
Comment (ljacomet) [inline subprojects/docs/src/docs/release/notes.md:334]: And I would add a general link to the documentation here.

**Review by ljacomet — CHANGES_REQUESTED** [review]
Thanks, I like the release notes.

But I believe the two remarks below would improve them further:

File: subprojects/docs/src/docs/release/notes.md (hunk 3 above)
Comment (ljacomet) [reply subprojects/docs/src/docs/release/notes.md:334]: And mention that these require version `1.2.0` of the plugin.

File: subprojects/docs/src/docs/release/notes.md [hunk 4]
```diff
@@ -330,6 +330,27 @@ In other types of projects, it will use the same version of Java used by the Gra
 
 For more details, see the [user manual](userguide/codenarc_plugin.html).
 
+### Plugin publishing to the Gradle Plugin Portal
+
+Version [1.2.0](https://plugins.gradle.org/plugin/com.gradle.plugin-publish/1.2.0) of the [Gradle Plugin Portal](https://plugins.gradle.org/) is now available.
```
Comment (ljacomet) [inline subprojects/docs/src/docs/release/notes.md:335]:
```suggestion
Version [1.2.0](https://plugins.gradle.org/plugin/com.gradle.plugin-publish/1.2.0) of the `com.gradle.plugin-publish` plugin is now available.
It is required to benefit from the features and bugfixes listed below.

See [documentation](userguide/publishing_gradle_plugins.html) for more details.
```

**Review by ljacomet — CHANGES_REQUESTED** [review]
Your release notes mixes the portal and the plugin.

See proposal:

**Comment by ljacomet** [comment]
@bot-gradle test and merge

**Pull request merged**